aws/src/integration/java/org/apache/iceberg/aws/AwsIntegTestUtil.java

```diff
@@ -47,6 +58,23 @@ public static String testBucketName() {
     return System.getenv("AWS_TEST_BUCKET");
   }
 
+  /**
+   * Get the environment variable AWS_TEST_ACCESS_POINT for a default bucket to use for testing
+   * @return bucket name
+   */
+  public static String testAccessPointName() {
+    return System.getenv("AWS_TEST_ACCESS_POINT");
+  }
+
+  /**
+   * Get AccessPointARN for a default bucket to use for testing
+   * @return access point arn
+   */
+  public static String testAccessPointARN() {
+    // format : arn:aws:s3:region:account-id:accesspoint/resource
+    return String.format("arn:aws:s3:%s:%s:accesspoint/%s", testRegion(), testAccountId(), testAccessPointName());
```

Comment:
partition is not always `aws`, should use `PartitionMetadata.of(region)` to get the partition value